tests/drivers/regulator/api/src/main.c

```diff
@@ -611,6 +611,58 @@ ZTEST(regulator_api, test_get_mode_not_implemented)
 	zassert_equal(ret, -ENOSYS);
 }
 
+ZTEST(regulator_api, test_set_active_discharge_not_implemented)
```

Comment:
please update `test_common_config` test as well, to cover correct capture of dt info